handlers/player.go

```diff
@@ -169,6 +172,19 @@ func (h *handler) setSynchronized(w http.ResponseWriter, r *http.Request, synchr
 		events.GroupID(sessionID),
 		playerctrl.SetSynchronizedPayload{UserID: userID, Synchronized: synchronized},
 	)
+
+	usr, err := h.UserCollection.GetUserByID(ctx, userID)
+	if err != nil {
+		handleError(w, http.StatusInternalServerError, log.ErrorLevel, msg, err, InternalServerError)
+		return
+	}
+
+	result := &struct {
+		Synchronized bool `json:"synchronized"`
+	}{
+		Synchronized: usr.SpotifySynchronized,
+	}
+	jsonResponse(w, result)
```

Comment:
publish the event after `GetUserByID`
this could lead to a bug since events are async